src/main/webapp/js/finscholar/scholarshiplistview/scholarshiplistview.js

```diff
@@ -15,21 +15,30 @@
 /** @fileoverview The mini controller for scholarship list view. */
 
 goog.module('finscholar.scholarshiplistview');
-const {scholarshiplist} = goog.require('finscholar.scholarshiplistview.templates');
+
+const {CommonListView} = goog.require('finscholar.commonlistview');
+const {ScholarshipListDataHandler} = goog.require('datahandlers.scholarshiplistdatahandler');
```

Comment:
Same thing here. There's a compile error here that's causing the rest of your branches to break. required namespace 'datahandlers.scholarshiplistdatahandler' is not defined. I'll re-approve your other PRs once this gets fixed.